constants/util.go

```diff
@@ -88,5 +92,21 @@ var (
 				CreateDir,
 			},
 		},
+		Flye: {
+			Name:             Flye,
+			DHubURL:          "dockerhub.io/nanozoo/flye", // https://github.com/fenderglass/Flye
+			OutputDir:        FlyeOut,
+			AssemblyFileName: "final.contigs.fa",
+			Comm: func(cfg *config_parser.Config) []string {
+				return []string{
+					fmt.Sprintf("--genome-size %s", cfg.Assemblers.Flye.GenomeSize),
+					fmt.Sprintf("--nano-raw %s", RawSeqIn),
```

Comment:
How about adding a check to see which type of raw data is provided and adding an appropriate parameter accordingly.

Check this out: https://github.com/genomagic/genomagic/blob/ts/hackseq/constants/util.go#L101-L106

Reply:
Fixed: 9d4fffb5c9872e0f26967cc0fbcee9b45b6b0b48